app/assets/stylesheets/components/_navbar.sass

```diff
@@ -0,0 +1,16 @@
+$Navbar-spacing-base: $Theme-spacing-base
+$Navbar-blue: Theme-color(color, header)
```

Comment:
Yes. Though, as discussed earlier, let's try to name variables using the approach "generic to specific", from the left to the right